frontend/src/components/Header/LoginDialogue.tsx

```diff
@@ -11,17 +11,42 @@ import styles from "./LoginDialogue.module.css";
 
 export default function FormDialog() {
   const [open, setOpen] = useState(false);
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
 
   const handleClickOpen = () => {
     setOpen(true);
   };
-  const handleLogin = () => {
+  const handleClose = () => {
     setOpen(false);
   };
-  const handleClose = () => {
+  const handleLogin = (e: any) => {
+    e.preventDefault();
+    let opts = {
+      'email': email,
+      'password': password
+    }
+    fetch('/login', {
+      method: 'post',
+      body: JSON.stringify(opts)
+    }).then(r => r.json())
+      .then(token => {
+        if (token.access_token){
+          alert(token.access_token)          
+        }
+        else {
+          alert("Please type in correct username/password")
+        }
+      });
     setOpen(false);
   };
+  const handleEmailChange = (e: any) => {
+    setEmail(e.target.value);
+  };
 
+  const handlePasswordChange = (e: any) => {
```

Comment:
avoid `any` type declarations

TypeScript already knows by type inference what the type of e should be, but if it's still bugging you, the real type should be `ChangeEvent<HTMLInputElement>`